packages/react-components/src/hooks/useTimeSeriesData/useTimeSeriesData.ts

```diff
@@ -27,6 +27,8 @@ const DEFAULT_SETTINGS: TimeSeriesDataRequestSettings = {
   fetchFromStartToEnd: true,
 };
 
+const DEFAULT_REFRESH_RATE = 5000;
```

Comment:
Nit: Consolidate types and constants across packages. There should only be one default refresh rate.